book/website/community-handbook/bookdash/bookdash-events.md

```diff
@@ -0,0 +1,142 @@
+
+This subchapter provides information on the events that are organised for the book dash attendees.
+These events include pre-event drop-in calls, Day-1 brainstorming session and group dinner, and Day-2 working day, which are explained below.
+
+## Pre-Event drop-in calls
+
+The core team members hosts 2 pre-event drop-in calls, minimum one week before the main event to ensure that all the participants are made familiar with project, the format of the book dash, and how they can be contributing through their involvement at the book dash.
+It is an opportunity for the organising team and the selected attendees to get to know each other and discuss any concerns that they may have prior to the book dash.
+
+Several participants often come to these calls with some plans and topics they would like to explore for their contributions in _The Turing Way_.
+These topics can be something that is either missing in the existing chapters or need to be updated in the project.
+We also discuss _The Turing Way_ as a project in general and what the core team is working on to support the development of its guide on reproducibility, ethics, project design, collaboration, communication and a meta-book on the project itself.
+Most importantly, the core team goes through the contribution guideline and explain how attendees can effectively use their time at the book dash.
+
+The template for shared-notes can be accessed in the community handbook {ref}`here<ch-bookdash-precall-template>`.
+You can read notes from the calls from the most recent book dash [here](https://hackmd.io/@malvikasharan/bookdash).
+
+## Day-1: brainstorming and dinner
+
+The invited participants join the core team for an introduction session a day before the book dash event.
+This is a chance to everyone to meet the participants and helpers informally, learn about each other and connect with specific members who plan to work on related topic during the book dash.
+
+![](./figures/book-dash-feb20-intro1.jpg)
+*Three of our participants, Heidi, Esther and Adina, exchanging local sweets and snacks from their home country at the beginning of the event (picture by Kirstie Whitaker)*
+
+In a dedicated hour, the core team members provide an overview of _The Turing way_ project and resources that are available for the participants for their book dash contributions.
+The next hours are dedicated for brainstorming in small groups, organising thoughts and ideas for their contributions the next day, share their plans with each other at the end of the session and identify their collaborators based on the similarity of their topics of interest.
+
+The day ends with a group dinner hosted by the core team to allow participants to relax after an intense discussion session and enjoy meals together in an informal setting.
+In between the meals, participants are invited to give lightning talks where they can share their personal stories and talk about a project that they’re either working on or are excited about.
+
```

Comment:
Suggest to add: 

"Lightning talks are short (normally 2 minutes long) and are given without slides."